docs/http-api.md

```diff
@@ -198,6 +198,25 @@ curl -s http://localhost:6060/priority | jsonpp
         }
     }
 ]
+
+## Cache delete
+
+```
+GET /ccache/delete
+POST /ccache/delete
+```
+
+* `X-Org-Id`: required
+* patterns: one or more query (glob) patterns. Use `**` to mean "all data" (full reset)
```

Comment:
might be worth it to mention that this is a list of patterns. otherwise users are likely to submit `"**"` instead of `["**"]`